frontend/src/features/projects/dispose/steps/ProjectDraftStep.tsx

```diff
@@ -32,7 +32,6 @@ const ProjectDraftStep = ({ isReadOnly, formikRef }: IStepProps) => {
       <Formik
         initialValues={draftFormValues}
         validationSchema={ProjectDraftStepYupSchema}
-        enableReinitialize={true}
```

Comment:
Due to some changes in this PR there is no need to re-initialize the form after initial load. This actually simplifies the workflow of these dispose workflows a lot